dagger-functions/src/test/resources/test_udf.py

```diff
@@ -0,0 +1,17 @@
+from pyflink.table import ScalarFunction, DataTypes
+from pyflink.table.udf import udf
+
+class TestUdf(ScalarFunction):
+
+    def __init__(self):
+        self.counter = None
+
+    def open(self, function_context):
+        mg = function_context.get_metric_group()
+        self.counter = mg.add_group("udf", "TestUdf").counter("value")
```

Comment:
what metric do we exactly want to emit? currently it will increase counter of "udf" metric on every call

Reply:
yes, counter is used. cant use gauge because of the bug.

Reply:
no I mean the name of the counter "udf" does not make sense, why not completely remove metric?

Reply:
you mean not adding metrics for the udf?

Reply:
ya lets remove adding metrics